ui/app/templates/components/dashboard/vault-configuration-details-card.hbs

```diff
@@ -0,0 +1,38 @@
+<h3 class="title is-4" data-test-configuration-details-title>Configuration details</h3>
+
+<Hds::Table @caption="Vault configuration details" class="is-border-spacing-revert">
```

Comment:
😍  yay HDS! Can you create a Jira ticket tracking that the `<Hds::Table>` component has been implemented?

Reply:
sounds good just created!